src/ToricVarieties/NormalToricVarieties/constructors.jl

```diff
@@ -463,6 +472,38 @@ function del_pezzo_surface(b::Int; set_attributes::Bool = true)
     end
     variety = NormalToricVariety(PolyhedralFan(fan_rays, cones; non_redundant = true))
     
+    # make standard choice for weights of the cox ring
+    if b == 1
+        set_attribute!(variety, :torusinvariant_cartier_divisor_group, free_abelian_group(4))
+        set_attribute!(variety, :torusinvariant_weil_divisor_group, free_abelian_group(4))
+        set_attribute!(variety, :class_group, free_abelian_group(2))
+        set_attribute!(variety, :picard_group, free_abelian_group(2))
+        weights = matrix(ZZ, [1 1; 1 1; 1 0; 0 -1])
+        set_attribute!(variety, :map_from_torusinvariant_weil_divisor_group_to_class_group, hom(torusinvariant_weil_divisor_group(variety), class_group(variety), weights))
+        set_attribute!(variety, :map_from_torusinvariant_cartier_divisor_group_to_picard_group, hom(torusinvariant_cartier_divisor_group(variety), picard_group(variety), weights))
+        set_attribute!(variety, :map_from_torusinvariant_cartier_divisor_group_to_torusinvariant_weil_divisor_group, hom(torusinvariant_cartier_divisor_group(variety), torusinvariant_weil_divisor_group(variety), identity_matrix(ZZ,4)))
+    end
+    if b == 2
+        set_attribute!(variety, :torusinvariant_cartier_divisor_group, free_abelian_group(5))
+        set_attribute!(variety, :torusinvariant_weil_divisor_group, free_abelian_group(5))
+        set_attribute!(variety, :class_group, free_abelian_group(3))
+        set_attribute!(variety, :picard_group, free_abelian_group(3))
```

Comment:
I think you shoult be able to do:
`set_attribute!(varienty, :torus=>1, :paul=>true, :free=>A)`

Reply:
Yes this can be done. I don't think it'll save much, but then perhaps it does... one would have to benchmark.

But these are slow constructors anyway, so  maybe not a reason to worry much